client/me/reauth-required/index.jsx

```diff
@@ -239,21 +228,61 @@ const ReauthRequired = createReactClass( {
 
 					{ this.renderSMSResendThrottled() }
 
-					<FormButtonsBar>
-						<FormButton
-							disabled={ this.state.validatingCode || ! this.preValidateAuthCode() }
-							onClick={ this.getClickHandler( 'Submit Validation Code on Reauth Required' ) }
-						>
-							{ this.props.translate( 'Verify' ) }
-						</FormButton>
-
-						{ this.renderSendSMSButton() }
-					</FormButtonsBar>
+					<FormButton
+						className="reauth-required__button"
+						disabled={ this.state.validatingCode || ! this.preValidateAuthCode() }
+						onClick={ this.getClickHandler( 'Submit Validation Code on Reauth Required' ) }
+					>
+						{ this.props.translate( 'Verify' ) }
+					</FormButton>
 				</form>
+			</Card>
+		);
+	},
+
+	renderSecurityKey() {
```

Comment:
It almost seems like this is unnecessarily in its own method as it's too short and just feels like unnecessary indirection.

Reply:
Removed this method in 2867a000fa9f136aa092c5d0a864937afca51ab1